Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+
+WORKDIR /app
+
+RUN pip install poetry
+
+COPY pyproject.toml ./
```

add 
```suggestion

COPY pyproject.toml ./
```
COPY pyproject.toml poetry.lock* ./